src/xclim/indices/stats.py

```diff
@@ -1053,6 +1059,18 @@ def reindex_time(_da: xr.DataArray, _da_ref: xr.DataArray, _group: str):  # nump
     # standardized_precipitation_index where values are not negative.
     probs = prob_of_zero + ((1 - prob_of_zero) * dist_probs)
 
+    # User-defined probability of zero values
+    if zero_inflated:
+        if callable(prob_zero_method):
+            zeros_prob = prob_zero_method(prob_of_zero)
+        elif prob_zero_method == "center":
+            zeros_prob = prob_of_zero / 2
```

Comment:
Ok, so the relation in the paper is: 

Original probability of zero

$$p_0 = \frac{n_{p=0}}{n+1}$$

and with the center of mass method: 

$$\bar{p}_0 = \frac{n_{p=0} + 1}{2(n+1)}$$

Currently, we don't have the unbiased definition with the n+1, we just compute $p_0 = n_{p=0} / n$
```python
prob_of_zero = da.groupby(group_handler).map(lambda x: (x == 0).sum("time") / x.notnull().sum("time"))
```

Whether we use $n$ or $n+1$ in the relations above will typically have a small impact, but for cases with only few dry days,using $n_{p=0}$ or $n_{p=0}+1$ in the center-of-mass relation would matter. Do you agree?

Should we always the unbiased definition in any case?

Reply:
All that to say that your `zeros_prob = prob_of_zero / 2` will currently correspond to an alternate definition

$$\bar{p}_0' = n_{p=0} / ( 2 n)$$

Reply:
I was about to say, let's just use the unbiased (with a breaking warning).

But spei.R library (https://github.com/sbegueria/SPEI/blob/master/R/spei.R) uses the 1/n and not 1/(n+1), and I think it's popular enough to warrant the fact that we should have an option to implement the same method
```R
      # Probability of zero (pze)
      if (distribution != "log-Logistic") {
        pze <- sum(x.mon == 0) / length(x.mon)
        x.mon <- x.mon[x.mon > 0]
      }
```

We can add the option unbiased = False by default.